admin.go

```diff
@@ -65,6 +72,15 @@ type ClusterAdmin interface {
 	// This operation is supported by brokers with version 0.11.0.0 or higher.
 	DeleteACL(filter AclFilter, validateOnly bool) ([]MatchingAcl, error)
 
+	// List the consumer groups available in the cluster.
+	ListConsumerGroups() (map[string]string, error)
+
+	// Describe some group IDs in the cluster.
+	DescribeConsumerGroup(group string) (*GroupDescription, error)
```

Comment:
Can you please update the comment, it is not very clear here.

Reply:
done